### ISSUE TITLE
[Docs] Update installation guides to use ng add @ngrx/package@latest

### ISSUE THREAD

**Issue by m00zi** [ngrx/platform#2604]
my angular version:
```
Angular CLI: 10.0.1
Node: 12.13.1
OS: darwin x64

Angular: 10.0.2
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.1000.1
@angular-devkit/build-angular     0.1000.1
@angular-devkit/build-optimizer   0.1000.1
@angular-devkit/build-webpack     0.1000.1
@angular-devkit/core              10.0.1
@angular-devkit/schematics        10.0.1
@angular/cli                      10.0.1
@ngtools/webpack                  10.0.1
@schematics/angular               10.0.1
@schematics/update                0.1000.1
rxjs                              6.5.5
typescript                        3.9.6
webpack                           4.43.0`
```

I have created new angular project, 

`ng new myapp --strict`

`cd myapp`

`ng add @ngrx/store`

then I got this error :

`Installing packages for tooling via npm.
Installed packages for tooling via npm.
The package that you are trying to add does not support schematics. You can try using a different version of the package or contact the package author to add ng-add support.`

would you please support ?

**Comment by timdeschryver** [comment]
Seems like it installs @ngrx/store version 1.2.1 for some reason 🙄

**Comment by brandonroberts** [comment]
Its an Angular CLI issue. You explicitly have to use `@ngrx/store@latest`. I opened an issue here about it awhile back https://github.com/angular/angular-cli/issues/17024

**Comment by m00zi** [comment]
Thanks Brandon, this works for me, do you have any expectation when the Angular team they will fix this bug ?

`ng add @ngrx/store@latest`

**Comment by alex-okrushko** [comment]
That issue was closed as "WAI" 🤔

**Comment by brandonroberts** [comment]
@m00zi Ihey aren't. We probably need to update the docs to use the `latest` tag with `ng add`.

**Comment by timdeschryver** [comment]
@m00zi do you want to update the [docs](https://github.com/ngrx/platform/blob/master/projects/ngrx.io/content/guide/store/install.md#L24) to add the latest tag?

**Comment by m00zi** [comment]
yes, it will be great

**Comment by brandonroberts** [comment]
Fixed #2629
